packages/api/src/platforms/vtex/resolvers/deleteProductReview.ts

```diff
@@ -0,0 +1,10 @@
+import type { Context } from '..'
+import type { MutationDeleteProductReviewArgs } from '../../../__generated__/schema'
+
+export const deleteProductReview = async (
+  _: any,
```

Comment:
If you know the type here, it's good to add. If you don't know the type, you could use `_: unknown`